overlord/snapstate/snapmgr.go

```diff
@@ -76,6 +77,11 @@ func (snapst *SnapState) Current() *snap.SideInfo {
 	return snapst.Sequence[n-1]
 }
 
+// DevMode returns true if the snap is installed in developer mode.
+func (snapst *SnapState) DevMode() bool {
+	return snapst.Flags&int(snappy.DeveloperMode) != 0
```

Comment:
Let's please define a local `DevMode` constant here in this file. DeveloperMode currently means something else, and I'd prefer to not to mix the two. This will also avoid the int conversion.